Dockerfile

```diff
@@ -104,9 +104,4 @@ ENV \
   ASSET_URL='' \
   STATIC_MESSAGE_URL=''
 
-RUN apk add --no-cache curl
-HEALTHCHECK \
```

Comment:
I strongly recommend to keep the `HEALTHCHECK`. It helps in a lot of use cases, and I'm not aware of any disadvantages.

For example, one wouldn't notice with a Docker-based dev setup that `digitransit-ui`'s web server actually (including the bundling it does) isn't working as intended; With the health check, this is reported to the person running `docker run`.